### ISSUE TITLE
Passwords are saved as plaintext

### ISSUE THREAD

**Issue by willsherwood** [dav-s/GradespeedScraper#5]
When "remember password" is checked, the password is saved in plaintext (64-bit encoded) with no encryption in dep.dat

**Comment by EpicDavi** [comment]
Yeah, I'm not really sure how to handle that. Do you have any recommendations?

**Comment by EpicDavi** [comment]
@willsherwood Wow, I did not know that the encoding was just 64-bit encoding. This could simplify the encoding/decoding.
